lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -167,6 +167,19 @@ class LatticeValue {
 };
 } // end anonymous namespace
 
+static llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
+                                     const LatticeValue &lattice) {
+  if (lattice.isUnknown()) {
+    return os << "<Unknown>";
+  } else if (lattice.isOverdefined()) {
+    return os << "<Overdefined>";
+  } else if (lattice.isInvalidValue()) {
+    return os << "<Invalid>";
+  } else {
+    return os << "<" << lattice.getConstant() << ">";
+  }
+}
+
```

Comment:
If this isn't used, it may be worth putting it behind an `LLVM_DEBUG` macro (which would require the associated header include).